.github/scripts/sync-issue-labels.js

```
@@ -0,0 +1,151 @@
+        console.log(`[sync-issue-labels] Skipping #${issueNumber}: reference points to a pull request.`);
+        continue;
+      }
+
+      const issueLabelNames = normalizeLabelNames(issueResponse?.data?.labels || []);
+      console.log(
+        `[sync-issue-labels] Issue #${issueNumber} labels: ${
+          issueLabelNames.length ? issueLabelNames.join(", ") : "(none)"
+        }`
+      );
+
+      issueLabelNames.forEach((label) => labelsFromIssues.add(label));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)